report/report.py

```diff
@@ -87,6 +93,12 @@ async def cmd_emergency(self, ctx: commands.Context, *, message: str = None):
             mod_pings = " ".join([i.mention for i in log.members if not i.bot])
         await log.send(content=mod_pings, embed=data)
 
+        report_reply = self.make_reporter_reply(ctx, message, True)
+        if ctx.channel.type == discord.ChannelType.private:
+            await ctx.send(embed=report_reply)
+        else:
+            await ctx.author.send(embed=report_reply)
```

Comment:
This needs some error handling. If a user doesn't allow DMs it should fail silently without spitting an error message
(redbot has a default error handler that'll throw a message in ctx.channel like `there was an error in the report command`  if errors aren't caught.

Reply:
May be a bit OTT but what about creating a temporary channel, readable only by the reporter and mods+, where a confirmation message is sent, and the reporter is mentioned? Could add a react for confirmation, triggering nukage of channel once confirmed, and a timeout of, say, 6 hours, after which the channel is automatically nuked.

Reply:
Personally not a fan of that from a UX POV. It seems like it would be annoying and OTT, personally

Reply:
@issy the error handling has been added.

@tigattack I agree with Issy. I think it's easy enough to follow up via DM if we need to carry on the conversation.

Reply:
Sounds good 👍